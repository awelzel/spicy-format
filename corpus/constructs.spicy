if (1)
    1;

if (1) {
1;
}

for (i in [1,2,3]) {
    print i;
}

for (i in [1,2,3])
print i;

throw   "yikes" ;
