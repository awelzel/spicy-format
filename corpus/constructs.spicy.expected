if (1)
    1;

if (1) {
    1;
}

for (i in [1, 2, 3]) {
    print i;
}

for (i in [1, 2, 3])
    print i;

switch (1) {
    case 1: "a";
    case 2:
        # Comment.
        "b";
    case 3, 4:
        "c";
}

throw "yikes";
